### ISSUE TITLE
Ability to adjust amount of time skipped when hitting the skip button on Android mobile. 

### ISSUE THREAD

**Issue by Gadetron** [Stremio/stremio-features#765]
Would be useful for skipping intros, so I don't have to click it like 7 times to get past them.

And possibly the ability to shrink the area for skipping so I don't skip every time I try to bring up the android nav bar via a swipe.

**Comment by charliesheen14** [comment]
You know the seek duration can be changed to 30s so you need to click 2 times instead of 7.

**Comment by Gadetron** [comment]
> You know the seek duration can be changed to 30s so you need to click 2 times instead of 7.

Not on Android, from what I see, zero options to adjust it

**Comment by charliesheen14** [comment]
Android tv ? Or android phone ?

**Comment by Gadetron** [comment]
> Android tv ? Or android phone ?

Well since my post says android mobile, I would figure you understand that it's the mobile app, not the tv

**Comment by charliesheen14** [comment]
Then use stremio web. To adjust those settings.

**Comment by charliesheen14** [comment]
Just trying to help here !

**Comment by Gadetron** [comment]
Yeah the web thing not practical for phone use, also it doesn't want to work due to re debrid stuff

**Comment by charliesheen14** [comment]
Weird that it isn't possible to change on android phone.
